backend/lobby/consumers.py

```diff
@@ -354,8 +358,26 @@ def buy_card_request_handler(self, data):
     def card_store_sweep_request_handler(self, data):
         username, room, game, state = reconstruct_game(data)
         if state.players.current_player.username == username:
+            cards_swept = state.deck_handler.get_top_three_cards()
```

Comment:
The get_top_three_cards method could be removed.
`cards_swept = state.deck_handler.cardstore.copy()`

Reply:
Awesome and appreciate the feedback, the code had a slight issue but it worked as 

cards_swept = state.deck_handler.store.copy()

will need your assistance as an additional approval to merge